src/blocks/digital-wallets/hooks.js

```diff
@@ -129,11 +130,30 @@ export function useGooglePay( payments, paymentRequest ) {
 		( async () => {
 			try {
 				const __googlePay = await payments.googlePay( paymentRequest );
-				await __googlePay.attach( googlePayRef.current, {
-					buttonColor: getSquareServerData().googlePayColor,
-					buttonSizeMode: 'fill',
-					buttonType: 'long',
-				} );
+				await __googlePay
+					.attach( googlePayRef.current, {
+						buttonColor: getSquareServerData().googlePayColor,
+						buttonSizeMode: 'fill',
+						buttonType: 'long',
+					} )
+					.then( () => {
+						// Append 'opens in a new window' to the aria-label of the Google Pay button.
+						const button =
+							googlePayRef.current.querySelector( 'button' );
+						if ( button ) {
+							const existingAriaLabel =
+								button.getAttribute( 'aria-label' ) ||
+								__( 'Buy with GPay', 'woocommerce-square' );
```

Comment:
Wondering if we should use the `getSquareServerData` function here instead of hardcoding this string in two places?

Reply:
handled in https://github.com/woocommerce/woocommerce-square/pull/403/commits/2b2850379e31d2adeda972af29c080d066a1405d